src/config.rs

```diff
@@ -133,6 +137,13 @@ impl Cfg {
 
         let settings_file = SettingsFile::new(rustup_dir.join("settings.toml"));
 
+        // Centralised file for multi-user systems to provide admin/distributor set initial values.
+        let fallback_settings = if cfg!(not(windows)) {
+            FallbackSettings::new(PathBuf::from(UNIX_FALLBACK_SETTINGS))?
+        } else {
+            None
+        };
+
```

Comment:
Would it be better to have fallback_settings always be present via `Default::default()`

Reply:
I'm torn
it makes a certain bit of code cleaner
on the other hand, it means we erase knowledge of whether a fallback file was supplied, which may be useful later

Reply:
I accept that the knowledge may be useful in the future, so let's not erase it for now.  It's a trivial refactor if we decide later it's worth the cleanup.